frlzjosh/project_management_system
# PMS-4 -- Set up VueRouter and Vuex

## Ready
Ready steady beddy ;)

## Description
some shit

## Steps for Testing
use vue devtools and verify the components are there and the state is set up with "test"
